packages/migration/src/mods/7.0.0/warnDetailsRowCheckIsSelectedRemoved.ts

```diff
@@ -0,0 +1,9 @@
+import { migration } from '../../migration';
+import { mod } from 'riceburn';
+import { renameJsxProp } from 'riceburn/lib/mods/renameJsxProp';
+
+export default migration('Deprecated DetailsRowCheck.isSelected has been removed. Use DetailsRowCheck.selected instead.', () => {
+  mod('**/*.ts?(x)').asTypescript((node, modder) => {
```

Comment:
Few notes, when this is run in `create-react-app` by default it seems to traverse `node_modules` which takes a fair amount of time. Once more are added we may see time adding up. Something to keep an eye out for and measure.

Reply:
Wonder if we should default to starting the search under `src` instead? @kenotron FYI